test_data/cloud-init/ubuntu/local/network-config

```diff
@@ -0,0 +1,8 @@
+version: 2
+ethernets:
+  ens3:
+    match:
+      macaddress: 12:34:56:78:90:ab
+    addresses:
+    - 192.168.2.2/24
```

Comment:
It would be better to match the default IP address range (like when you do `--net tap=`)

Reply:
PTAL